client/src/components/VersionedBadge/VersionedBadge.scss

```diff
@@ -0,0 +1,16 @@
+.versioned-badge {
+  padding-left: 0;
+  padding-right: 0;
+
+  &--draft {
+    color: $state-draft;
+  }
+
+  &--modified {
+    color: $state-modified;
+  }
+
+  &--live {
+    color: $green;
```

Comment:
Did you forget to push something? I'm expecting this to be `$state-live` or `$state-published` or something, with that and the archived variable being added to variables.scss

Reply:
How embarrassing ... should be fixed now.